autoreject/ransac.py

```diff
@@ -184,7 +184,8 @@ def _compute_correlations(self, data):
 
     def fit(self, epochs):
         self.picks = _handle_picks(info=epochs.info, picks=self.picks)
-        _check_data(epochs, picks=self.picks, verbose=self.verbose)
+        _check_data(epochs, picks=self.picks,
+                    ch_constraint='single_channel_type', verbose=self.verbose)
```

Comment:
technically `ransac` could also support multiple channel types :P maybe for someone else ...